### PR TITLE
[doc, master] update intranet detection instructions

### PR DESCRIPTION
**Pull request by jwkaltz**
For issue https://github.com/camptocamp/c2cgeoportal/issues/3220

### CODE REVIEW

**Comment by jwkaltz** [comment]
The 2.1 doc PR should not have been merged into master, the instructions are not the same for master. I will rebase.

File: doc/integrator/intranet.rst [hunk 1]
```diff
@@ -33,11 +33,12 @@ This page lists the changes that must be applied to add such a functionality.
...
                return None
 
-           get_user_from_request = _create_get_user_from_request(request.registry.settings)
+           get_user_from_request = create_get_user_from_request(request.registry.settings)
            user = get_user_from_request(request)
 
            if user is None and request.environ.get('intranet', 0) == '1':
-               from c2cgeoportal_commons.models import DBSession, Role
+               from c2cgeoportal_commons.models import DBSession
+               from c2cgeoportal_commons.models import Role
```
Comment (sbrunner) [inline doc/integrator/intranet.rst]: Why this change?

File: doc/integrator/intranet.rst [hunk 2]
```diff
@@ -69,7 +70,19 @@ This page lists the changes that must be applied to add such a functionality.
...
+
+   .. code:: apache
+
+       SetEnvIf REMOTE_ADDR ${intranet_ip} intranet=1
+
+   or, depending on the proxies setup:
+
+   .. code:: apache
+
+       SetEnvIf x-forwarded-for ${intranet_ip} intranet=1
```
Comment (sbrunner) [inline doc/integrator/intranet.rst]: The 2.3 doc is on docker than we don't have any apache folder ...

File: doc/integrator/intranet.rst (hunk 1 above)
Comment (jwkaltz) [reply doc/integrator/intranet.rst]: I thought this was our preferred style... but actually, I see we have both styles so I will undo this change.

File: doc/integrator/intranet.rst (hunk 2 above)
Comment (jwkaltz) [reply doc/integrator/intranet.rst]: I am not sure I understand. We still have an apache/wsgi.conf.mako (in ./geoportal/c2cgeoportal_geoportal/scaffolds/nondockercreate), so if somebody uses this, this change to the configuration is still necessary?

Comment (sbrunner) [reply doc/integrator/intranet.rst]: Yes but I think the preferable to have the documentation for the Docker version or specify that's only fro the non Docker version :-)

Comment (jwkaltz) [reply doc/integrator/intranet.rst]: OK

**Comment by jwkaltz** [comment]
Jenkins fails. I can not see any details (I can not login).  AFAIK Jenkins is actually not required here, but I see no way to get rid of it in this PR. So I close this PR and will try a new one.
